mix.exs

```diff
@@ -23,7 +23,7 @@ defmodule TDS.Ecto.Mixfile do
     [
       {:ecto, "~> 1.0.0"},
       {:tds, "~> 0.5.1"},
-      {:poison, only: :test}
+      {:poison, "~> 1.0"}
```

Comment:
Yeah, lets not require Poison, we can clean it up later to operate better with mix. I'll merge this in then and release a new version. Thanks for the PR!!! :bowtie: